.github/actionlint.yaml

```diff
@@ -13,16 +16,22 @@ self-hosted-runner:
     - linux.8xlarge.nvidia.gpu
     - linux.16xlarge.nvidia.gpu
     - linux.g5.4xlarge.nvidia.gpu
+    # Repo-specific IBM hosted S390x runner
     - linux.s390x
+    # Organization wide AWS Windows runners
     - windows.4xlarge.nonephemeral
     - windows.8xlarge.nvidia.gpu
     - windows.8xlarge.nvidia.gpu.nonephemeral
     - windows.g5.4xlarge.nvidia.gpu
-    - bm-runner
+    # Repo-specific AMD hosted MI300 runners
     - linux.rocm.gpu
+    # Repo-specific Apple hosted M2 runners
+    - macos-m2-metal
```

Comment:
Do you plan to rename or add the `macos-m2-metal` label here?  I see that it has `macos-m2-14` label atm, and I mistakenly thought that it was a typo with our `macos-m1-14` runner in https://github.com/pytorch/pytorch/pull/127582.  I'm going to put the runner back now with the correct label